lib/wallet/http.js

```diff
@@ -1046,8 +1046,8 @@ class HTTP extends Server {
       const sign = valid.bool('sign', true);
 
       assert(name, 'Name is required.');
-      assert(bid, 'Bid is required.');
-      assert(lockup, 'Lockup is required.');
+      assert(bid !== null, 'Bid is required.');
```

Comment:
nit: should we use type coercion here, i.e. `bid != null`? It will protect against the case where`bid` is `undefined`.

Reply:
I don't think `bval` returns `undefined` in any case. And I'm pretty sure it only returns `false` specifically if the type is `bool`.

Reply:
I think you're right though, will update